hack/verify-staticcheck.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+REPO_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+GOLANGCI_LINT_VER="v1.57.1"
+
+cd "${REPO_ROOT}"
+source "hack/util.sh"
+
+if util::cmd_exist golangci-lint ; then
+  echo "Using golangci-lint version:"
+  golangci-lint version
+else
+  echo "Installing golangci-lint ${GOLANGCI_LINT_VER}"
+  # https://golangci-lint.run/usage/install/#other-ci
+  curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(go env GOPATH)/bin ${GOLANGCI_LINT_VER}
```

Comment:
I don't highly recommend using the code from the `master` branch, you should use a fixed version, such as `v1.57.1` that you defined above.

Reply:
done